lib/gds-components/header/Header.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * GDS component: https://design-system.service.gov.uk/components/header/
+ */
+
+export interface HeaderProps {
+  homeURL: string;
+}
+
+export const Header = ({ homeURL }: HeaderProps) => (
+  <header className="govuk-header" role="banner" data-module="govuk-header">
+    <div className="govuk-header__container govuk-width-container">
+      <div className="govuk-header__logo">
+        <a
```

Comment:
Techically we should be using Link components within react-router but there are two reasons I'm not doing that at the moment:

- I don't want to have react-router-dom imports in the lib folder as the routing is only configured from src
- The behaviour is a little different, the link highlighting doesn't go away because the page isn't reloading

Even if using an anchor tag loses state, that's not an issue because they'd be going right back to the beginning of the journey anyway